lib/Packager/module-resolver.es6

```diff
@@ -375,8 +375,8 @@ function EXPAND (target, log, cache, opts, cwd) {
 		if(result.json) {
 			let moduleDir = result.json.moduleDir;
 			if (!moduleDir) {
-				let libDir = path.join(opts.cwd, opts.libDir, dir);
-				if (libDir === step) {
+				let libDir = path.resolve(step, '..')
```

Comment:
Not at all haha! But was glancing at emails and saw this and it caught my eye ;)